staging/src/k8s.io/client-go/pkg/api/helpers.go

```diff
@@ -465,6 +465,44 @@ const (
 	ObjectTTLAnnotationKey string = "node.alpha.kubernetes.io/ttl"
 )
 
+// AddOrUpdateTolerationInPod tries to add a toleration to the pod's toleration list.
+// Returns true if something was updated, false otherwise.
+func AddOrUpdateTolerationInPod(pod *Pod, toleration *Toleration) (bool, error) {
```

Comment:
According to the `returns`, it'll never return error; so we will not reject any pods. Should we check whether "something was updated" ? 

/cc @davidopp @kevin-wangzefeng @gmarek @ncdc